src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java

```diff
@@ -336,4 +335,8 @@ private Properties baseProducerProperties() {
         props.put("bootstrap.servers", getKafkaBrokerConnect());
         return props;
     }
+
+    interface RetryAction {
+        void run() throws ExecutionException, InterruptedException, KafkaException, TimeoutException;
+    }
```

Comment:
Oh, very good.  We should have complained the first time we saw as much duplicated!